webpack.common.js

```diff
@@ -9,9 +8,32 @@ module.exports = {
         new HtmlWebpackPlugin({
             template: path.resolve(__dirname, "src", "index.html"),
         }),
-        new MiniCssExtractPlugin(),
         new MonacoWebpackPlugin({
-            languages: ["ascii", "ini"],
+            features: [
+                "!suggest",
+                "!folding",
+                "!gotoSymbol",
+                "!linesOperations",
+                "!multicursor",
+                "!hover",
+                "!indentation",
+                "!codelens",
+                "!linkedEditing",
+                "!rename",
+                "!smartSelect",
```

Comment:
I believe `smartSelect` refers to the ability to expand / contract selection based on code syntax / logics. Since we display logs rather than code, we might not need the feature. 

I tried re-enabling `smartSelect` and tested with http://localhost:3010/?filePath=https://yscope.s3.us-east-2.amazonaws.com/sample-logs/yarn-ubuntu-resourcemanager-ip-172-31-17-135.log.1.clp.zst#logEventIdx=28 with the "Pretty" function and saw no significant difference in the selection behaviour when double / triple clicking.